mdm/checkin_event.go

```diff
@@ -60,6 +63,17 @@ type userTokenUpdate struct {
 	NotOnConsole  bool   `plist:",omitempty"`
 }
 
+// Get Bootstrap Token Message Type # GetBootstrapTokenRequest
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
I updated these to match existing code. For example `auth struct` has the comment `// Authenticate Message Type`, not `auth Message Type`. These are private structs so they're not going to show up in godoc anyways. If you want it to actually be `//getBootstrap ...` I can do that too.